installing miscellaneous package fails

```
pkg install -forge miscellaneous
```
```
configure: error: The program GNU Units is required to install Octave-Forge miscellaneous package.
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking term.h usability... yes
checking term.h presence... yes
checking for term.h... yes
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking for units... no

pkg: error running the configure script for miscellaneous.
error: called from
    configure_make at line 82 column 9
    install at line 184 column 7
    pkg at line 441 column 9
```

--- a/org.octave.Octave.yaml
+++ b/org.octave.Octave.yaml
@@ -34,6 +34,7 @@ cleanup:
 - /bin/qhull
 - /bin/qvoronoi
 - /bin/rbox
+- /com
 - /examples
 - /include/QSci
 - /lib/cmake
@@ -48,6 +49,7 @@ cleanup:
 - /share/fltk
 - /share/hdf5_examples
 - /share/man
+- /share/units/locale_map.txt
 - fluid
 - fluid.*
 build-options:
@@ -301,6 +303,14 @@ modules:
     sha256: 77774b3f4a06c20705cc2ef1c804864422e3cf95235e965b1f00a46df7da5f62
   - type: patch
     path: texinfo-perl-528.patch
+
+- name: units # dependency of the miscellaneous package
+  sources:
+  - type: archive
+    url: https://ftpmirror.gnu.org/gnu/units/units-2.19.tar.gz
+    sha256: 4262136bdfc152b63ff5a9b93a7d80ce18b5e8bebdcffddc932dda769e306556
+  - type: patch
+    path: units-install-info.patch
 
 - name: zeromq # dependency of the zeromq package
   buildsystem: cmake-ninja

--- a/org.octave.Octave.yaml
+++ b/org.octave.Octave.yaml
@@ -313,6 +313,8 @@ modules:
     sha256: 4262136bdfc152b63ff5a9b93a7d80ce18b5e8bebdcffddc932dda769e306556
   - type: patch
     path: units-install-info.patch
+  post-install:
+  - mv -f /app/com/units/currency.units /app/share/units/currency.units
 
 - name: zeromq # dependency of the zeromq package
   buildsystem: cmake-ninja

--- a/org.octave.Octave.yaml
+++ b/org.octave.Octave.yaml
@@ -305,6 +305,8 @@ modules:
     path: texinfo-perl-528.patch
 
 - name: units # dependency of the miscellaneous package
+  config-opts:
+  - PYTHON=/usr/bin/python3
   sources:
   - type: archive
     url: https://ftpmirror.gnu.org/gnu/units/units-2.19.tar.gz

--- a/units-install-info.patch
+++ b/units-install-info.patch
@@ -1,0 +1,12 @@
+diff --git a/Makefile.in b/Makefile.in
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -167,7 +167,7 @@ install-info: units.info
+	if test -f $(DESTDIR)$(infodir)/dir; then \
+	  if $(SHELL) -c 'install-info --version' >/dev/null 2>&1; then \
+	    install-info --dir-file=$(DESTDIR)$(infodir)/dir \
+-	               $(DESTDIR)$(infodir)/units.info; \
++	               $(DESTDIR)$(infodir)/units.info || :; \
+	  else true; fi \
+	else true; fi
+ 

